metrics/web/test/features/auth/presentation/widgets/auth_input_field.dart

```diff
@@ -0,0 +1,176 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/features/auth/presentation/widgets/auth_input_field.dart';
+
+void main() {
+  const String testLabel = 'testLabel';
+
+  group("AuthInputField", () {
+    testWidgets("displays the label", (WidgetTester tester) async {
+      await tester.pumpWidget(const _AuthInputFieldTestbed(
+        label: testLabel,
+      ));
+
+      expect(find.widgetWithText(AuthInputField, testLabel), findsOneWidget);
+    });
+
+    testWidgets("delegates a controller to the text input widget",
+        (WidgetTester tester) async {
+      final controller = TextEditingController();
+      await tester.pumpWidget(_AuthInputFieldTestbed(
+        controller: controller,
+      ));
+
+      final textInput = tester.widget<TextField>(find.byType(TextField));
+
+      expect(textInput.controller.hashCode, equals(controller.hashCode));
```

Comment:
```suggestion
      expect(textInput.controller, equals(controller));
```
I think we can use the default `==` operator here.